### PR TITLE
Tech Group page for OK Meetups

### LINKED ISSUE #41: Other meetup groups
Add a page that lists other tech groups in Oklahoma. Include any from this list that seem relevant: https://www.meetup.com/pro/Techlahoma/

Other groups to consider including:
https://www.meetup.com/CODE-Norman
https://www.meetup.com/Data-Creativity-OKC
https://www.meetup.com/DSN-DEV-OKC
https://www.facebook.com/groups/free.code.camp.tulsa/
https://www.facebook.com/groups/492652307741260/

### PR DESCRIPTION
**Pull request by mattstub**
Have added a basic page where visitors can find other Oklahoma based tech groups across the state, its broke up into 3 subcategories (Statewide, OKC, Tulsa). This page is linked to in the about page.

This PR resolves #41.

### CODE REVIEW

**Review by Jason9199 — CHANGES_REQUESTED** [review]
I think the new page looks great! I would suggest maybe rewording the new sentence on the about.html page to say "You can find more Tech Related groups throughout Oklahoma to suit your interests here." As it stands right now it has an extra period in it.

**Comment by mattstub** [comment]
revised the wording, after digesting it a couple times it wasn't reading out very well. also added a link to Jack's github repo but still left the link commented out at this time as I don't believe he is an active organizer.

**Review by Jason9199 — APPROVED** [review]
Excellent! Thank you for the contribution.
